Makefile

```diff
@@ -0,0 +1,51 @@
+all: setup build test lint
```

Comment:
Feel free to not add this file.

In a simple single-package repo, I will often not set up a `Makefile` until I concretely need it to do something. I'm happy to document `stack this` and `stack that` in the README, and (unlikely previously) GitHub Actions won't be re-using make tasks. Therefore, the utility of it is reduced, IMO.

It looks like you copied from `bcp47` which has one both because it was vestigial and because it's a multi-package repo with a semi-complicated release step.

Reply:
Gone!